erpnext_shipping/erpnext_shipping/doctype/sendcloud/sendcloud.py

```diff
@@ -224,42 +336,14 @@ def get_carrier(self, carrier_name, post_or_get=None):
 		else:
 			return carrier_name.upper() if post_or_get == "get" else carrier_name.lower()
 
-	def get_parcel_dict(
-		self,
-		shipment,
-		parcel,
-		index,
-		delivery_company_name,
-		delivery_address,
-		delivery_contact,
-		service_info,
-		description_of_content,
-		value_of_goods,
-	):
+	def get_parcel(self, parcel, shipment, index, description_of_content, value_of_goods):
 		return {
-			"name": f"{delivery_contact.first_name} {delivery_contact.last_name}",
-			"company_name": delivery_company_name or delivery_address.address_title,
-			"address": delivery_address.address_line1,
-			"address_2": delivery_address.address_line2 or "",
-			"city": delivery_address.city,
-			"postal_code": delivery_address.pincode,
-			"telephone": delivery_contact.phone,
-			"request_label": True,
-			"email": delivery_contact.email_id,
-			"data": [],
-			"country": delivery_address.country_code.upper(),
-			"shipment": {"id": service_info["service_id"]},
+			"dimensions": {
+				"length": parcel.get("length", 0),
+				"width": parcel.get("width", 0),
+				"height": parcel.get("height", 0),
+				"unit": "cm",
+			},
+			"weight": {"value": parcel.get("weight", 0), "unit": "kg"},
```

Comment:
The weight was previously rounded to the number of decimals allowed by sendcloud. Is this no longer needed?

https://github.com/frappe/erpnext-shipping/pull/24